contracts/ConstantPriceFeed.sol

```diff
@@ -0,0 +1,61 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.15;
+
+import "./vendor/@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+contract ConstantPriceFeed is AggregatorV3Interface {
+    /** Custom errors **/
+    error InvalidInt256();
+    error NotImplemented();
+
+    /// @notice Version of the price feed
+    uint public constant override version = 1;
+
+    /// @notice Description of the price feed
+    string public constant description = "Constant price feed";
+
+    /// @notice Number of decimals for returned prices
+    uint8 public immutable override decimals;
+
+    /// @notice The constant price
+    int private immutable CONSTANT_PRICE;
+
+    /**
+     * @notice Construct a new scaling price feed
+     * @param decimals_ The number of decimals for the returned prices
+     **/
+    constructor(uint8 decimals_) {
+        decimals = decimals_;
+        CONSTANT_PRICE = signed256(10 ** decimals_);
+    }
+
+    /**
+     * @notice Unimplemented function required to fulfill AggregatorV3Interface; always reverts
+     **/
+    function getRoundData(uint80 _roundId) override external pure returns (uint80, int256, uint256, uint256, uint80) {
+        revert NotImplemented();
```

Comment:
I wonder if it makes more sense to just define our own *actual* interface at this point, otherwise this begs the question a little of the purpose of the interface since we are basically just tricking the compiler

Reply:
Do you mean a separate interface only for this price feed? That works, but `Comet` code does use the `AggregatorV3Interface`. Either way, calling this specific function selector on this price feed would revert, but at least this price feed currently conforms to the interface that `Comet` expects.

Reply:
I mean change the interface Comet actually uses (which is a subset of aggregator), and then it wouldn't depend on vendor anymore (solidity change, but shouldn't affect the bytecode), and we could reference that here

Reply:
I see, that makes sense. We do lose the ability to verify it against the vendor repo using Vendoza though, if you are okay with that.

Reply:
What are we actually verifying in this case?

Reply:
We are verifying the `AggregatorV3Interface` we use in this repo against the interface defined in the Chainlink repo. More of an extra sanity check.

Reply:
But does that add anything? We use the interface we use right, and we sim against the actual contracts, whether or not its defined in Chainlink repo doesn't even give us extra sanity, right?

Reply:
Practically, it offers very little extra sanity as you said because of scenarios. But scenarios also don't have 100% coverage of all edge-cases, so extra sanity checks isn't a bad thing.

That being said, I'm totally okay with not having this interface be verified by Vendoza (I wasn't defending it, rather just pointing out we lose its checks). 

However, I do want to discuss whether or not changing `Comet`'s `AggregatorV3Interface` is a worthwhile change. The `ConstantPriceFeed` is really the only price feed that lacks a functioning `getRoundData`. All other price feeds that Comet interacts with, including the other 2 custom feeds in this PR, will have a functioning `getRoundData`. We could make the `ConstantPriceFeed` use a different interface, but I'm not sure it's worthwhile to change the interface that `Comet` uses just because of this one price feed for the WETH deployment. IMO, it seems more correct to be using the full canonical Chainlink interface.

Reply:
But we never *use* `getRoundData` even if it's there, right? It's weird to me that we would have an interface that has functions that we don't rely on, and then revert in impl bc we know we don't actually use the fn for anything. I don't feel strongly but maybe should discuss live, I don't see what's more correct about it

Reply:
Synced offline. I created a separate price feed interface without `getRoundData` to be used by `Comet` and extended by these wrapper price feed contracts. This new interface prevents us from accidentally calling an unsupported function (e.g. `getRoundData` for `ConstantPriceFeed`) in any future changes.